bin/deepforge

```diff
@@ -243,6 +243,15 @@ program.command('start')
     .option('-N, --no-conda', 'do not start deepforge server in a conda environment.')
     .option('-m, --mongo', 'start MongoDB')
     .action(async args => {
+        if(args.conda){
+            try {
+                Conda.check();
+            } catch (e) {
+                console.log('Warning: option to use a conda is selected but conda installation ' +
```

Comment:
There is probably a better way to phrase this. Perhaps something like "Warning: conda executable not found. Please install conda for automatic management of Python dependencies via virtual environments."

Reply:
I have added the following message:
```
Warning: conda executable not found. Please install conda for automatic management of Python dependencies via conda environments.
```